### PR TITLE
chatrequest updated model and schema

### PR DESCRIPTION
**Pull request by ElenaPegoraro**
Hello, I updated the logic based on enum for chatrequest model and schema, by adding a status (pendind, rejected, confirmed), please check it up.

### CODE REVIEW

File: db/migrate/20220906104711_revome_accepted_from_chat_requests.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class RevomeAcceptedFromChatRequests < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :chat_requests, :accepted, :booelan
```
Comment (croab) [inline db/migrate/20220906104711_revome_accepted_from_chat_requests.rb:3]: Check spelling of boolean :)

**Review by croab — CHANGES_REQUESTED** [review]
Just one typo needs correcting

File: db/migrate/20220906104711_revome_accepted_from_chat_requests.rb (hunk 1 above)
Comment (EleoXDA) [reply db/migrate/20220906104711_revome_accepted_from_chat_requests.rb:3]: Nice catch ))

Comment (ElenaPegoraro) [reply db/migrate/20220906104711_revome_accepted_from_chat_requests.rb:3]: Ok thank you for the catch

**Review by croab — APPROVED** [review]

**Pull request merged**